src/main/java/com/checkmarx/flow/controller/ADOController.java

```diff
@@ -61,6 +63,7 @@
         validateBasicAuth(auth);
         controllerRequest = ensureNotNull(controllerRequest);
         adoDetailsRequest = ensureDetailsNotNull(adoDetailsRequest);
+        ResourceContainers resourceContainers = body.getResourceContainers();
```

Comment:
These lines and lines  190-193 seems duplicated. Is it possible to extract them to a common logic?